src/webui/src/static/function.ts

```diff
@@ -45,19 +45,30 @@ function parseMetrics(metricData: string): any {
     }
 }
 
+const isArrayType = (list: any): boolean | undefined => {
+
+    if (!Array.isArray) {
```

Comment:
Looks like these are browser compatibility stuff. I thought TS or node should deal with them?

Reply:
had updated.